src/main/resources/messages.yml

```diff
@@ -98,6 +98,9 @@ info.money: '&6Balance: &b{0}'
 info.ally: '&6Allies: &b{0}'
 info.online: '&a'
 info.offline: '&c'
+info.onlinePlayer: '<green>{0}</green>'
+info.offlinePlayer: '<red>{0}</red>'
```

Comment:
would this not break things on servers that do not have adventuretext enabled for whatever reason?